internal/distro/rhel82/distro_test.go

```diff
@@ -93,3 +96,327 @@ func TestFilenameFromType(t *testing.T) {
 		})
 	}
 }
+
+func TestImageType_BuildPackages(t *testing.T) {
+	x8664BuildPackages := []string{
+		"dnf",
+		"dosfstools",
+		"e2fsprogs",
+		"grub2-pc",
+		"policycoreutils",
+		"qemu-img",
+		"systemd",
+		"tar",
+		"xz",
+	}
+	aarch64BuildPackages := []string{
+		"dnf",
+		"dosfstools",
+		"e2fsprogs",
+		"policycoreutils",
+		"qemu-img",
+		"systemd",
+		"tar",
+		"xz",
+	}
+	buildPackages := map[string][]string{
+		"x86_64":  x8664BuildPackages,
+		"aarch64": aarch64BuildPackages,
+	}
+	d := rhel82.New()
+	for _, archLabel := range d.ListArches() {
+		archStruct, err := d.GetArch(archLabel)
+		if err != nil {
```

Comment:
You can use assert.NoError or require.NoError everywhere where comparing `err`

Reply:
I think not in the places with the `continue` statement, but thanks for pointing this out, I fixed the asserts in the remaining occurrences.

Reply:
assert returns a boolean and takes care of logging the issue for you so you can do
```
if assert.NoError(err, "...") {
    continue
}
``

Reply:
I see... fixed :)